chain/ethereum/src/codec.rs

```diff
@@ -134,7 +134,10 @@ impl Into<web3::types::U64> for TransactionTraceStatus {
 impl Into<Option<web3::types::U64>> for TransactionTraceStatus {
     fn into(self) -> Option<web3::types::U64> {
         match self {
-            Self::Unknown => None,
+            Self::Unknown => {
+                {}
```

Comment:
is this needed?

Reply:
It's a separate fix that can go in another PR, we will extract it to it's own PR.

Reply:
See #3138, @sduchesneau need to rebase once my other PR is merged.

Reply:
Merged, ready for rebase